### DISCUSSION TITLE
prime ng version v-16.7.1 issue

### DISCUSSION THREAD

**Discussion by ghost** [primefaces/community#826] [PrimeNG]
when in p-multiselect i enable filter option and there is not search result found that message is not coming. any workaround how to solve it

**Comment by cetincakiroglu** [comment]
Hi,

Could you please test it on v17, multiselect received a lot of updates and fixes between v16.7 and v17.3

**Reply by ghost** [reply]
hi @cetincakiroglu after updating the its working but other issue are getting there

**Reply by cetincakiroglu** [reply]
Could you please test it in the `latest` version of PrimeNG.
